zebra/zebra_vxlan.c

```diff
@@ -4360,10 +4359,9 @@ void zebra_vxlan_remote_vtep_add(ZAPI_HANDLER_ARGS)
 			continue;
 		}
 
-		zif = ifp->info;
 
 		/* If down or not mapped to a bridge, we're done. */
-		if (!if_is_operative(ifp) || !zif->brslave_info.br_if)
```

Comment:
We can't unconditionally drop this dependency. For cases where the the BD is represented by a {bridge, vlan} this may create problems including invalid memory access based on the sequence of events.

Reply:
Would the correct thing to do here be to check the interface type, and if it is a type that should depend on bridge membership (let's say all interface types except for VTEPs to be conservative) then retain this condition, otherwise drop the dependency?